components/concept/engine/src/main/java/mozilla/components/concept/engine/prompt/PromptRequest.kt

```diff
@@ -6,15 +6,22 @@ package mozilla.components.concept.engine.prompt
 
 import android.content.Context
 import android.net.Uri
-import mozilla.components.concept.storage.Login
 import mozilla.components.concept.engine.prompt.PromptRequest.Authentication.Level
 import mozilla.components.concept.engine.prompt.PromptRequest.Authentication.Method
 import mozilla.components.concept.engine.prompt.PromptRequest.TimeSelection.Type
+import mozilla.components.concept.storage.Login
+import java.util.UUID
 
 /**
  * Value type that represents a request for showing a native dialog for prompt web content.
+ *
+ * @param shouldDismissOnLoad Whether or not the dialog should automatically be dismissed when a new page is loaded.
+ * @param uid [PromptRequest] unique identifier. Ignored when comparing if [PromptRequest] subclasses are equals.
  */
```

Comment:
Should we add a not about: Subclass not tacking into consideration `shouldDismissOnLoad` and `uid` for equals?

Reply:
Sure.
Thinking of updating that comment block to something like:
```
/**
 * Value type that represents a request for showing a native dialog for prompt web content.
 *
 * @param shouldDismissOnLoad Whether or not the dialog should automatically be dismissed when a new page is loaded.
 * Defaults to `true`.
 * @param uid [PromptRequest] unique identifier. Defaults to a random UUID.
 * (This two parameters, though present in all subclasses are not evaluated in subclasses equals() calls)
 */
 ```

Reply:
👍🏽